client/state/ui/guided-tours/actions.js

```diff
@@ -46,7 +45,7 @@ export function nextGuidedTourStep( { tour, stepName } ) {
 
 	const trackEvent = recordTracksEvent( 'calypso_guided_tours_next_step', {
 		step: stepName,
-		tour_version: guidedToursConfig.get( tour ).meta.version,
+		//tour_version: guidedToursConfig.get( tour ).meta.version,
```

Comment:
Do we want to leave this as is for this PR? I remember seeing discussions on how to handle this, but didn't see the decision.